runtime/browser/xenon-base.js

```diff
@@ -0,0 +1,80 @@
+let XenonStaterMixin = Base => class extends Base {
```

Comment:
there is quite a lot of commented-out code in this file - is it code that we'll need in the future, or maybe it is possible to remove it?